crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -248,6 +248,25 @@ module deepbook::clob_v2 {
         quote_asset_trading_fees: Balance<QuoteAsset>,
     }
 
+    /// Capability granting permission to access an entry in `Pool.quote_asset_trading_fees`.
```

Comment:
I think we need a comment here about when these were added +that older pools (maybe we can just list out the mainnet object ID's if there are few enough) do not have a `PoolOwnerCap`.